packages/component/src/Middleware/AttachmentForScreenReader/createCoreMiddleware.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import AudioAttachment from './AudioAttachment';
+import FileAttachment from './FileAttachment';
+import ImageAttachment from './ImageAttachment';
+import TextAttachment from './TextAttachment';
+import VideoAttachment from './VideoAttachment';
+
+export default function createCoreMiddleware() {
+  return () => next => (...args) => {
+    const [
+      { activity: { from: { role } = {} } = {}, attachment, attachment: { contentType, contentUrl, thumbnailUrl } = {} }
+    ] = args;
+
+    return role === 'user' && !/^text\//u.test(contentType) && !thumbnailUrl
```

Comment:
Why user here?

Reply:
(This is same logic as in `packages/component/src/Middleware/Attachment/createCoreMiddleware.js`)

For example, audio attachments sent by the user, should not be rendered as `<audio>`.

In general, attachments sent by the user, except text and image, should rendered as a file.